bug-pattern/src/main/java/jp/skypencil/findbugs/slf4j/ManualGetStackTraceDetector.java

```diff
@@ -0,0 +1,36 @@
+package jp.skypencil.findbugs.slf4j;
+
+import com.google.common.base.Objects;
+import edu.umd.cs.findbugs.BugReporter;
+import edu.umd.cs.findbugs.OpcodeStack.Item;
+import jp.skypencil.findbugs.slf4j.parameter.AbstractDetectorForParameterArray2;
+
+/**
+ * FindBugs (now SpotBugs) Detector for (ab)use of {@link Throwable#getStackTrace()} in SFL4j logger.
+ *
+ * @author Michael Vorburger.ch
+ */
+public class ManualGetStackTraceDetector extends AbstractDetectorForParameterArray2 {
+
+    @Item.SpecialKind
+    private final int isGetStackTrace = Item.defineNewSpecialKind("use of throwable getStackTrace");
```

Comment:
`isXxx` is common naming for boolean, better to rename...

Reply:
done